src/amo/components/AddonMeta.js

```diff
@@ -66,6 +79,13 @@ export class AddonMetaBase extends React.Component {
   }
 }
 
+export const mapStateToProps = (state) => {
+  return {
+    userId: state.user ? state.user.id : null,
```

Comment:
Due to the beauty of Redux, `state.user` will never be falsy. You can just do `userId: state.user.id` -- `id` will be null when the user is not signed in.